src/proc/colorizer.h

```diff
@@ -15,6 +13,19 @@ namespace rs2
 
 namespace librealsense {
 
+    // Enhancement for debug mode that incurs performance penalty with STL
```

Comment:
what's wrong with `std::clamp`?

Reply:
Oh sorry it's for C++17, but I still don't see the benefit of the `_DEBUG` version of this function, any compiler with minimal optimization enabled will create the same binary for both versions of this function...
In addition:
- The function is not `constexpr` which could improve its performance (given `consexpr` input
- `T`s are passed by value instead of `const&` which adds performance penalty

Reply:
I'll switch to const&.
The '_DEBUG' switch does the opposite - the enhancement is for non-optimized builds only